src/gui/qgsexpressionpreviewwidget.h

```diff
@@ -17,6 +17,7 @@
 #define QGSEXPRESSIONPREVIEWWIDGET_H
 
 #include <QWidget>
+#include <QAction>
```

Comment:
this can be a forward declaration, no need to include

Reply:
Updated.